plugins/colorsource/plugin.json

```diff
@@ -0,0 +1,92 @@
+    "Downloads (including Personality Editor)": "https://www.etcconnect.com/Products/Consoles/ColorSource/Software.aspx",
+    "Console Manual": "https://www.etcconnect.com/WorkArea/DownloadAsset.aspx?id=10737496502",
+    "Personality Editor manual": "https://www.etcconnect.com/WorkArea/DownloadAsset.aspx?id=10737490177"
+  },
+  "fixtureUsage": [
+    "<p>",
+    "Load the <code>userlib.jlib</code> onto the root of a USB drive and plug it into the console",
+    "(or into your computer, if you use ColorSource Offline).",
+    "When patching devices, select \"User Library\" instead of \"Standard Library\".",
+    "</p>"
+  ],
+  "fileLocations": {
+    "subDirectoriesAllowed": false,
+    "USB drive (for console)": {
+      "user": "/userlib.jlib"
+    }
+  },
+  "additionalInfo": [
+    "<h3>ColorSource channel types</h3>",
+    "",
+    "<table>",
+    "<tr>",
+    "  <th scope=\"col\">CS type</th>",
+    "  <th scope=\"col\">Description</th>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>0</code></td>",
+    "  <td>No function / Hidden</td>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>1</code></td>",
+    "  <td>Intensity</td>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>2</code></td>",
+    "  <td>Pan, Tilt, PanTiltSpeed</td>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>3</code></td>",
+    "  <td>Color wheel, ColorPreset, ColorTemperature</td>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>4</code></td>",
+    "  <td>\"Beam\" – basically everything that doesn't fit to another type</td>",
+    "</tr>",
+    "<tr>",
+    "  <td><code>5</code></td>",
+    "  <td>Single Color (see below), Hue, Saturation</td>",
+    "</tr>",
+    "</table>",
+    "",
+    "<h3>Known colors</h3>",
+    "<p>Red, Green, Blue, White (~7500K or unknown temperature), Amber, UV, ",
+    "Cyan, Magenta, Yellow, Indigo, Lime, Orange, RedOrange, MintGreen, GreenCyan, ",
+    "CoolWhite (~9000K), MediumWhite (~4000K), WarmWhite (~3200K)</p>",
+    "<p>Extracted from <code>C:/Program Files (x86)/ETC/ColorSource/Library/ColorTable.idx</code>.</p>",
+    "",
+    "",
+    "<h3>Features not supported by the Personality Editor</h3>",
+    "",
+    "<h4>Commands</h4>",
+    "<p>",
+    "Commands define a sequence of specific DMX values with hold time to perform a certain action, e. g. device reset.",
+    "We could (in theory) use this feature to make use of our Maintenance.hold property.",
+    "However, commands are used in internal fixtures only, the .jlib importer seems to simply delete the commands property.",
+    "Nevertheless, we include the commands property in our exported fixtures as the logic to create them has already been implemented and it may be supported in future.",
+    "</p>",
+    "",
+    "<h4>Range tables</h4>",
+    "",
+    "<p>",
+    "Range tables provide a way of defining switching channels. They aren't supported in .jlib files yet and are preserved for internal fixtures only.",
```

Comment:
Please wrap long lines like you've done above.

Reply:
You've only done this for the last of these lines; the other ones are also very long (*Comment on lines 82 to 89*).

Also, make sure to add a space at the end of a line, since the strings are concatenated without a separator.